A mess of new features and fixes (#20)
aquach/vim-mediawiki-editor: README.md

## Usage

### Mappings

vim-mediawiki-editor has a few default bindings. They should probably change 
to something smarter.

`gf` navigates to the wikilink under your cursor.

`C-w f` opens the wikilink under your cursor in a new window.

`gl` opens the backlinks for the wikilink under your cursor.

> `g.` opens the backlinks for the article you are on.

Reviewer: Maybe word this a little better?
```suggestion
`g.` opens the backlinks for the currently loaded article.
```
